modules/cnv-monitoring-upgrade-status.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/upgrading-container-native-virtualization.adoc
+
+[id="cnv-monitoring-upgrade-status_{context}"]
+= Monitoring upgrade status
+
+The best way to monitor {CNVProductName} upgrade status is to watch the
+ClusterServiceVersion (CSV) `PHASE`.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+. Run the following command:
++
+----
+$ oc get csv
+----
+
+. Review the output, checking the `PHASE` field. For example:
++
+----
+VERSION  REPLACES                                        PHASE
+2.1.1    kubevirt-hyperconverged-operator.v2.1.0         Installing
+2.1.0                                                    Replacing
+----
+
+. (Optional) Monitor the aggregated status of all {CNVProductName} component
+`Conditions` by running the following command:
++
+----
+$ oc get hco -n openshift-cnv hyperconverged-cluster \
+-o=jsonpath='{range .status.conditions[*]}{.type}{"\t"}{.status}{"\t"}{.message}{"\n"}{end}'
+----
++
+A successful upgrade results in the following output:
++
+----
+ReconcileComplete  True  Reconcile completed successfully
+Available          True  Reconcile completed successfully
+Progressing        False Reconcile completed successfully
+Degraded           False Reconcile completed successfully
+Upgradeable        True  Reconcile completed successfully
+----
+
```

Comment:
may it stay as it's now. We give that command to monitor the status of the upgrade. The given output will be displayed only when the upgrade is finished, as the statement above says.

Reply:
As long as the note's nested the right way, yes.

Reply:
I actually intended for it to *not* be connected to the command, because `PHASE` refers to the first part of the procedure, and `Conditions` refers to the second, optional part. Do you think that makes sense?